llvm/test/Feature/Repo/repo_InsertOrExtraValueInst_hash.ll

```diff
@@ -3,13 +3,14 @@
 ;
 ; RUN: rm -f %t.db
 ; RUN: env REPOFILE=%t.db opt -S %s -o %t
-; RUN: env REPOFILE=%t.db opt -S %t | FileCheck %s
+; RUN: env REPOFILE=%t.db llc -mtriple="x86_64-pc-linux-gnu-repo" -filetype=obj %t -o %t.1.o
+; RUN: env REPOFILE=%t.db opt -S %s | FileCheck %s
 
 target triple = "x86_64-pc-linux-gnu-repo"
 
-@.str = private unnamed_addr constant [9 x i8] c"test.cpp\00", align 1
+@.str = unnamed_addr constant [9 x i8] c"test.cpp\00", align 1
 
-define internal void @__cxx_global_var_init() section ".text.startup" personality i8* bitcast (i32 (...)* @__gxx_personality_v0 to i8*) {
+define void @__cxx_global_var_init() section ".text.startup" personality i8* bitcast (i32 (...)* @__gxx_personality_v0 to i8*) {
```

Comment:
Is this change to prevent DCE  from removing these two names?

Reply:
Yes, you are right. If they are internal, DCE will remove them.